terminal/terminal.go

```diff
@@ -37,12 +38,21 @@ func New(client service.Client, conf *config.Config) *Term {
 	if conf != nil && conf.Aliases != nil {
 		cmds.Merge(conf.Aliases)
 	}
+
+	dumb := false
```

Comment:
Let's move the dumb terminal check here, and then based on that we can assign `w` to either `os.Stdout` directly (for the `dumb == true` case) or the result of `getColorableWriter()`. That prevents us from having to do that check in multiple places, and in terminal_other we can just always return `os.Stdout`, and on windows we go through the rest of the logic you've defined.